.travis.yml

```diff
@@ -1,16 +1,27 @@
 language: go
 
+# cache:
+#   directories:
+#   - $HOME/.glide
+
 go:
 - 1.7.1
 
 branches:
   only:
   - master
   - cluster-up
+  - /^issue-[0-9]+$/
 
 install:
+# Work around the fact that I don't seem to be able to change the git checkout command
+# In order to build any fork, we need to make sure that the code lives in 'minishift'
```

Comment:
What do you mean `build any fork`? This is for this repo only & should be checked out correctly in GOPATH by travis.